modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -190,6 +191,12 @@ inline void copyFromIE(const IE::Blob::Ptr &blob, MatType &mat) {
     }
 }
 
+inline int getIdxByName(const std::vector<std::string>& reshape_names, const std::string& name) {
+    const auto it = std::find(reshape_names.begin(), reshape_names.end(), name);
+    GAPI_Assert(it != reshape_names.end());
```

Comment:
Consider to put `error` message. `GAPI_Assert(it != reshape_names.end() && "Message")`